Source/Core/DolphinWX/Debugger/RegisterView.cpp

```diff
@@ -301,6 +301,9 @@ wxString CRegTable::GetValue(int row, int col)
       if (row < 16)
         return wxString::Format("IBAT%01d", row - 8);
 
+      if(row == 16)
```

Comment:
Missing space after `if`

Reply:
Fixed. Thanks for the thorough style check.